docs/src/app/components/Card/README.md

```diff
@@ -0,0 +1,12 @@
+## Card
+
+A card is a piece of paper with unique related data that serves as an  
```

Comment:
Please provide a link to https://www.google.com/design/spec/components/cards.html.